src/components/Select/Select.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import styled from "styled-components"
+import theme from "../../utils/theme"
+import { common } from "../button/Button"
+
+const Select = styled.select`
+  ${common}
+  border: 1px solid ${theme.primary};
+  color: ${theme.primary};
+  padding: 10px 8px;
+  position: relative;
+
+  &:hover:not(:disabled) {
+    background: ${theme.primary};
+    color: ${theme.bg};
+  }
+
+  & option {
+    background: ${theme.bg};
+    color: ${theme.primary};
+    margin: 0;
+    &:hover {
+      background-color: ${theme.primary};
+      color: ${theme.bg};
+    }
+  }
+`
+
+interface Props
+  extends React.DetailedHTMLProps<
+    React.SelectHTMLAttributes<HTMLSelectElement>,
+    HTMLSelectElement
+  > {}
+
+export default function (props: Props): React.ReactElement {
```

Comment:
It's my mistake, but always when you have option avoid default exports. They can make you fragile to typos, and are problematic with hints for intelisense. Also for refactors there are really bad because it's hard to find all refs in project and rename.

This practice is part of legacy bad code :/. Can we replace it with export const { Select } ?

Reply:
OK, corrected